fabtests/pytest/efa/efa_common.py

```diff
@@ -113,3 +113,12 @@ def get_efa_domain_names(server_id):
 
     return efa_domain_names
 
+def get_efa_device_names(server_id):
+    domain_names = get_efa_domain_names(server_id)
+    device_names = set()
+    for domain_name in domain_names:
+        itemlist = domain_name.split("-")
+        assert len(itemlist)==2
```

Comment:
nit: space

Reply:
we should be running flake8 formatter on these files too.

Reply:
added